src/main/java/com/fasterxml/jackson/databind/introspect/POJOPropertiesCollector.java

```diff
@@ -554,8 +554,10 @@ protected void _addFields(Map<String, POJOPropertyBuilder> props)
                         _anySetterField = new LinkedList<>();
                     }
                     _anySetterField.add(f);
+                    // 20-July-2024: [databind#4396]: Skip the rest of processing, but only
```

Comment:
Need to change to refer to this PR (#4755) or maybe issue, #4396 is the closed PR.